client/src/components/Modal/Modal.css

```diff
@@ -0,0 +1,125 @@
+.Modal {
+  width: 100%;
+  height: 309px;
+  margin: 0 auto;
+  padding: 1rem 2rem 0.5rem 2rem;
+  background: var(--yellow);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 3px;
+  display: flex;
+  flex-direction: column;
+  font-family: var(--default-font);
+  font-weight: var(--title-weight);
+  color: black;
+  transition: all 0.8s;
+}
+
+.Modal__close {
+  text-align: right;
+}
+
+.Modal__closeButton {
+  font-size: 5rem;
+  line-height: 56px;
+  text-decoration: none;
+  color: black;
+}
+
+.Modal__body {
+  width: 100%;
+  height: 60%;
+}
+
+.Modal__messageBox {
+  width: 90%;
+  height: 100%;
+  margin: 0 auto;
+}
+
+.Modal__messageText {
+  text-align: center;
+  font-size: 38px;
+  line-height: 57px;
+}
+
+.Modal__footer {
+  width: 100%;
+  height: 23%;
+  display: flex;
+  flex-direction: row;
+}
+
+.Modal__footerButtons {
+  width: 100%;
+}
+
+.Modal__footerButtonConfirm {
```

Comment:
Does the button component not work here? It's okay if it doesn't but that would be ideal

Reply:
From what I can tell, the styling for these buttons are quite specific (different color, borders, sizes) from what is prescribed in the storybook buttons, and they appear to be standalone